internal/handlers/common/error.go

```diff
@@ -25,7 +25,7 @@ import (
 
 //go:generate ../../../bin/stringer -linecomment -type ErrorCode
 
-// ErrorCode represents wire protocol error code.
+// ErrorCode represents FerretDB error codes.
```

Comment:
I think it makes it less clear. Everything in the FerretDB code base is about FerretDB. And the value of that type represents a single error code, not many error codes.

Reply:
reverted.